src/test_linked_list.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding utf-8 -*-
+import pytest
+
+
+def test_push():
+    from linked_list import LinkedList
+    ll = LinkedList()
+    ll.push('bob')
+    assert ll.head.get_data() == 'bob'
```

Comment:
What other effects are there of pushing? If you add data to a linked list its size should go up!